src/Microsoft.Health.Fhir.Shared.Api/Features/Exceptions/BaseExceptionMiddleware.cs

```diff
@@ -26,27 +27,25 @@ public class BaseExceptionMiddleware
         private readonly RequestDelegate _next;
         private readonly ILogger<BaseExceptionMiddleware> _logger;
         private readonly IFhirRequestContextAccessor _fhirRequestContextAccessor;
-        private readonly CorrelationIdProvider _correlationIdProvider;
-        private readonly IContentTypeService _contentTypeService;
+        private readonly IFormatParametersValidator _parametersValidator;
 
         public BaseExceptionMiddleware(
             RequestDelegate next,
             ILogger<BaseExceptionMiddleware> logger,
             IFhirRequestContextAccessor fhirRequestContextAccessor,
             CorrelationIdProvider correlationIdProvider,
```

Comment:
Since the class variable for correlation id provider has been removed the constructor parameter isn't needed anymore.

Reply:
Good catch, wonder why VS didn't tell me that.

---
In reply to: [587683450](https://github.com/microsoft/fhir-server/pull/1699#discussion_r587683450) [](ancestors = 587683450)